src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -103,6 +103,12 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
      */
     public const PRESERVE_EMPTY_OBJECTS = 'preserve_empty_objects';
 
+    /**
+     * allow to overide json format with elasticsearch format (it is json)
```

Comment:
`Allow to override [...]`

But this description is very specific, not sure it's appropriate.
I'll let @dunglas do the remaining of the review :)